ddtrace/contrib/django/patch.py

```diff
@@ -0,0 +1,22 @@
+import wrapt
+
+import django
+
+def patch():
+    """Patch the instrumented methods
+    """
+    if getattr(django, '_datadog_patch', False):
+        return
+    setattr(django, '_datadog_patch', True)
+
+    _w = wrapt.wrap_function_wrapper
+    _w('django', 'setup', traced_setup)
+
+def traced_setup(wrapped, instance, args, kwargs):
+    from django.conf import settings
+
+    settings.INSTALLED_APPS.append('ddtrace.contrib.django')
+    settings.MIDDLEWARE_CLASSES.insert(0, 'ddtrace.contrib.django.TraceMiddleware')
```

Comment:
I think we should add the `hasattr` also here because it's deprecated in Django 1.10 so it will be surely removed starting from the next release. In this way we're relatively "safe" for the Django 1.11 release.